client/my-sites/email/titan-add-mailboxes/index.jsx

```diff
@@ -285,13 +285,13 @@ class TitanAddMailboxes extends Component {
 					) }
 
 					{ selectedDomain && titanMailProduct && hasTitanMailWithUs( selectedDomain ) && (
-						<MailboxPricingNotice
+						<EmailPricingNotice
 							domain={ selectedDomain }
 							expiryDate={ getTitanExpiryDate( selectedDomain ) }
 							mailboxRenewalCost={ getTitanMailboxRenewalCost( selectedDomain ) }
 							mailboxPurchaseCost={ getTitanMailboxPurchaseCost( selectedDomain ) }
 							product={ titanMailProduct }
-							isMonthlyProduct={ isTitanMonthlyProduct( titanMailProduct ) }
+							isMonthlyBilling={ isTitanMonthlyProduct( titanMailProduct ) }
```

Comment:
Was this an existing bug? (It looks like it! 😬 )

Reply:
Yes it was 🙈.